sdks/python/apache_beam/transforms/display.py

```diff
@@ -137,9 +137,19 @@ def create_payload(dd):
       except ValueError:
         # Skip if the display data is invalid.
         return None
-      if 'value' not in display_data_dict or 'label' not in display_data_dict:
+
+      if 'key' not in display_data_dict or 'value' not in display_data_dict:
+        # Skipping DisplayData items that do not have a 'key' or 'value' set.
```

Comment:
Should we assert that value is always present? Otherwise, WDYT about making `value = None` if it is not defined?

Reply:
Ah, there is validation already here: 
https://github.com/chamikaramj/beam/blob/5921a75c9166d524e3c0362a6c59837dcba15da5/sdks/python/apache_beam/transforms/display.py#L304-L327

Reply:
So, I think we can just omit this `if` ?

Reply:
Yeah. Removed.